tensorboard/plugins/scalar/vz_line_chart/vz-line-chart.html

```diff
@@ -35,15 +35,10 @@
     <div id="tooltip">
       <table>
         <thead>
-          <tr>
-            <th><!-- run color swatch --></th>
-            <th>Name</th>
-            <th>Smoothed</th>
-            <th>Value</th>
-            <th>Step</th>
-            <th>Time</th>
-            <th>Relative</th>
-          </tr>
+          <!-- We inject the HTML instead of using dom-repeat because polymer
+               does not support dom-repeat templates within table elements. -->
```

Comment:
Wow. I'm disappointed, but not surprised. I suppose that this is the best bad alternative?